modules/prebidServerBidAdapter/index.js

```diff
@@ -268,10 +268,9 @@ function _appendSiteAppDevice(request, pageUrl) {
     request.app = config.getConfig('app');
     request.app.publisher = {id: _s2sConfig.accountId}
   } else {
-    request.site = {
-      publisher: { id: _s2sConfig.accountId },
-      page: pageUrl
-    }
+    request.site = (typeof config.getConfig('site') === 'object') ? config.getConfig('site') : {};
+    request.site.publisher = { id: _s2sConfig.accountId },
+    request.site.page = pageUrl;
```

Comment:
So if someone sets `site` in setConfig, we want to use it.

But we want to overwrite `publisher` and `page` no matter what?

Here is what the publisher object specifies in openRTB protocol:

![image](https://user-images.githubusercontent.com/13972794/78702095-ea669e80-78bc-11ea-96c1-7979881c2359.png)

So if we did:

```
pbjs.setConfig({
   site: {
      ...
      publisher: {
         id: 'some id'
         domain: 'prebid.org'
         ...
      }
});
```

Then we would basically not include the other keys in the user defined publisher object.

Maybe it makes more sense to use `utils.deepSet` in order to set the `id` of publisher, so that if `publisher` already exists we do not overwrite the entire thing but just the `id` key?

Reply:
Please add a test for this as well